installer/client/.eslintrc.js

```diff
@@ -18,4 +18,12 @@ module.exports = {
     "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
     "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
   },
+  overrides: [
+    {
+      files: ["*.vue"],
+      rules: {
+        "vue/multi-word-component-names": "off",
```

Comment:
:thinking:

Reply:
this rule is not allowing to have single word as component name. i think it will introduce unnecessary verbosity